### ISSUE TITLE
[experimental] anywidget.experimental.command called without self

### ISSUE THREAD

**Issue by keller-mark** [manzt/anywidget#543]
Not sure a bug or an intended API change, however i did not see it in the release notes.

In 0.9.6, the `@anywidget.experimental.command` functions are called with 3 arguments `self`, `msg`, `buffers`
In 0.9.7, the `@anywidget.experimental.command` functions are called with 2 arguments `msg`, `buffers`

```py
import anywidget
class Widget(anywidget.AnyWidget):
      _esm = """
      export default {
        async render({ model, el, experimental }) {
          let [msg, buffers] = await experimental.invoke("_echo", "hello, world");
          console.log(msg); // "HELLO, WORLD"
        },
      };
      """

      @anywidget.experimental.command
      def _echo(self, msg, buffers):
          # upper case the message
          return msg.upper(), buffers
Widget()
```
For example results in TypeError in 0.9.7
```
---------------------------------------------------------------------------

TypeError                                 Traceback (most recent call last)

[/usr/local/lib/python3.10/dist-packages/anywidget/experimental.py](https://localhost:8080/#) in handle_anywidget_command(self, msg, buffers)
    157             return
    158         cmd = cmds[msg["name"]]
--> 159         response, buffers = cmd(msg["msg"], buffers)
    160         self.send(
    161             {

TypeError: Widget._echo() missing 1 required positional argument: 'buffers'
```

https://colab.research.google.com/drive/1IdUTo7vu3WqCDJcDn6TaZsIwWW9owzFP?usp=sharing

In the case of the VitessceWidget and ZarrWidget, this does not allow accessing `self._store` / keeping a Zarr store on `self`

**Comment by manzt** [comment]
Thanks for reporting. This is a bug and will be reverted to the 0.9.6 version. I'll make a fix

**Comment by manzt** [comment]
Fixed in v0.9.8
